src/app/components/BalanceBox/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useAccount } from "~/app/context/AccountContext";
+
+type Props = {
+  className?: string;
+};
+
+function BalanceBox(props: Props) {
+  const { balancesDecorated } = useAccount();
+  const balanceParts = balancesDecorated.accountBalance.split(" ");
+
+  return (
+    <div
+      className="w-full flex flex-col items-center justify-center dark:text-white p-4"
```

Comment:
I noticed the className property from outside is not applied, as there is no host element. I tested and I think you have to replace the className line with

```
className={`w-full flex flex-col items-center justify-center dark:text-white p-4 ${props.className}`}
```

Reply:
@reneaaron what do you think about this ?

Reply:
@11WALTER11 Rene mentioned to me somewhere I can just as well update this part in the PR where I need it. So you don't need to care about it.